frontend/src/store/controller/reducers/backend.ts

```diff
@@ -128,7 +135,12 @@ export const eventLogReducer = (
     }
     case BACKEND_CONNECTION_DOWN: {
```

Comment:
It may be helpful to add a comment that this reducer will behave in unwanted ways (e.g. creating multiple ephemeral events) if `BACKEND_CONNECTION_DOWN` is dispatched multiple times after the connection goes down.

An alternative approach could be to look up events inside the state's `ephemeralLogEvents` to see if there's already a `frontend_backend_connection_down` event, but perhaps that adds complexity which isn't worth it for now. You could also mention this other feasible approach in the code comment, for future reference.